samples/delete_dataset.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
We recently updated all the license headers to be consistent.  Could I trouble you to copy from an existing template?